tools/mod/go.mod

```diff
@@ -1,6 +1,6 @@
 module go.etcd.io/etcd/tools/v3
 
-go 1.22
```

Comment:
This has happened to me a couple of times, but I usually manually revert to `go 1.22`. 

Do you happen to know why this is done automatically? And if we should keep this? :)

Reply:
> Do you happen to know why this is done automatically? And if we should keep this? :)

Because at least one of dependencies depends on 1.22.4 in their go.mod files. We should find out the dependencies and request them to make change in the similar way as etcd does.

Reply:
Just curious, is there any benefit to using `go 1.22` instead of `go 1.22.x`?

Reply:
I'm unsure if there's an easy way of tracing the dependency that locks Go's version up to the patch level. However, from the current updates [github.com/sivchari/tenv](https://github.com/sivchari/tenv/blob/main/go.mod#L3) does specify `1.21.0`.

Reply:
We have no choice unless we don't bump the dependencies which depend on go1.22.4. Let's keep it as it's for now. 




> Just curious, is there any benefit to using `go 1.22` instead of `go 1.22.x`?

We should try not to push other applications, which depend on etcd sdk, to bump their go versions. The `tools` should be fine.